Client/mods/deathmatch/CClient.cpp

```diff
@@ -13,6 +13,7 @@
 #define ALLOC_STATS_MODULE_NAME "client"
 #include "SharedUtil.hpp"
 #include <core/CClientCommands.h>
+#include <../Client/core/CClientChatboxVariables.h>
```

Comment:
The header belongs in `Client/sdk/core` if you want to interface between core and deathmatch.